modules/porous_flow/include/actions/PorousFlowAddBCAction.h

```diff
@@ -0,0 +1,28 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "MooseObjectAction.h"
+
+/**
+ * Action that sets up BCs for porous flow module
```

Comment:
Did you intend that this Action should be used for other BCs too?  If not, then change to "Action that adds BCs corresponding to hot/cold fluid injection, viz a PorousFlowSink and a PorousFlowEnthalpySink"

Reply:
> Did you intend that this Action should be used for other BCs too?

Yes. That's my plan. If there are other BCs like this one, we just add it here, so it is all nicely in one place :-)